blocks/cards/cards.js

```diff
@@ -0,0 +1,87 @@
+import {
+  parseFragment,
+  removeEmptyElements,
+  render,
+  convertExcelDate,
+  timeSince,
+} from '../../scripts/scripts.js';
+
+const gdPlusIcon = '<svg width="51" height="19" viewBox="0 0 51 19" fill="none" xmlns="http://www.w3.org/2000/svg" role="img" class="gd-plus-logo" aria-label="GD Plus Logo" aria-hidden="true"><path class="gd-text" d="M12.231 17.6625C11.8994 17.8282 11.2644 18.0211 10.7676 18.0763V18.464C14.0812 18.3806 16.4007 17.8292 17.8932 17.1663V10.5952H12.231V17.6625Z" ></path><path class="gd-text" d="M17.4505 4.85311V1.15447C16.2639 0.82303 14.3307 0.60256 12.2871 0.573V0.96065C14.3855 1.95545 15.8766 3.08689 17.4505 4.85311Z" ></path><path class="gd-text" d="M7.37138 9.5175C7.37138 4.30023 8.94531 1.53921 11.21 0.96016V0.57251C5.02429 0.98632 0.9375 4.24402 0.9375 9.5165C0.9375 14.6528 4.55459 18.1838 10.0224 18.4605V18.0728C8.03449 17.7162 7.37138 14.5138 7.37138 9.5175Z" ></path><path class="gd-text" d="M27.4187 0.823H21.1777V18.2143H27.4187V0.823Z" ></path><path class="gd-text" d="M28.3848 0.823V1.23778C30.8429 1.45825 32.0019 3.16972 32.0019 9.5189C32.0019 15.5919 30.79 17.6338 28.4148 17.8276V18.2153C34.6277 18.1324 38.4386 15.2062 38.4386 9.5199C38.4367 3.30782 34.4876 0.90634 28.3848 0.823Z" ></path><path d="M46.6875 5.22485H44.8125V13.8499H46.6875V5.22485Z" fill="#ED3E49"></path><path d="M41.4375 8.5999V10.4749H50.0625V8.5999H41.4375Z" fill="#ED3E49"></path></svg>'; // prettier-ignore
```

Comment:
@DriftingSands  I believe cards are rendered below the fold so instead of rendering the SVG directly in the HTML, you can store it as image and reference it as image with the attribute loading="lazy" so that it only loads once the user scrolls into it.